docs/src/concepts/channels/index.md

```diff
@@ -0,0 +1,12 @@
+# Channels
+
+`fuelup` adopts a simplified version of `rustup` [channels](https://rust-lang.github.io/rustup/concepts/channels.html). Currently, only the `latest` channel will be published and serve as a source of distribution of Fuel toolchain binaries.
+
+| Channel      | Source          | Integration Tested   | Update Frequency         | Available |
+| ------------ | --------------- | -------------------- | ------------------------ | --------- |
+| **[latest]** | published bins  | ✔️                   | checked every 30 minutes | ✔️        |
```

Comment:
Why is `latest` in square brackets? It's not a link?

Reply:
Ah, it's meant to link to the latest channel on the next page but it was broken before.

https://github.com/FuelLabs/fuelup/pull/134/files#diff-b9e0f260d64132ce19cdd88e9a284bfbf76c2b2eed0056d9f5faca98bb4a82a8R12

 Thanks for the catch!